client/data/authorizations/resolvers.ts

```diff
@@ -52,10 +54,32 @@ export function* getAuthorizations( query: Query ): Generator< unknown > {
 export function* getAuthorization(
 	paymentIntentId: string
 ): Generator< unknown > {
-	// TODO this is implemented in other PR: https://github.com/Automattic/woocommerce-payments/pull/4688/
-	yield updateAuthorization( {
-		payment_intent_id: paymentIntentId,
-	} as Authorization );
+	try {
+		if ( paymentIntentId ) {
+			const result = yield apiFetch( {
+				path: `${ NAMESPACE }/authorizations/${ paymentIntentId }`,
+			} );
+			const isCaptured = result
+				? ( result as GetAuthorizationApiResponse ).is_captured
```

Comment:
Should we assign a variable to `result as GetAuthorizationApiResponse` as it's being use twice.

Reply:
I refactored things a bit here [b6186c2](https://github.com/Automattic/woocommerce-payments/pull/5006/commits/b6186c211081b8c2245b838ae546da56aca8075c)